tests/acceptance/features/coreApiSharePublicLink1/accessToPublicLinkShare.feature

```diff
@@ -1,13 +1,16 @@
 @api
 Feature: accessing a public link share
+  As a public
+  I want to check the preview of a public link file
+  So that I know I have access to it
```

Comment:
```suggestion
  As a person who knows a public link
  I want be able to access the preview of a public link file
  So that I can see a small preview of the file before deciding to download it
```

Maybe something like this is the real reason for being able to "see" previews?

Reply:
Yes this is better